edgelet/contrib/config/linux/config.yaml

```diff
@@ -73,6 +73,10 @@
 #                       provision with DPS, and only fall back to the provisioning backup
 #                       from a previous successful provisioning if the attempt fails.
 #
+#                       If setting this to false, it's recommended to also set
```

Comment:
Do we have a doc bug around  always_provision_on_startup and the recommendation around dynamic provisioning? 
If not, please open one and we should work with Kelly to update together with the release.

Reply:
I'll check with Varun and Anca. I'm not sure if we have docs for `dynamic_reprovisioning` either.